src/styles/blocks/stars.css

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url('../../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Since `.stars__star` elements are laid out horizontally, consider applying margin only to the right or left to maintain consistency and to avoid potential margin collapse issues.